Documentation/coding-guidelines/project-guidelines.md

```diff
@@ -56,28 +56,50 @@ Each project will define a set of supported build configurations
 ```
 <PropertyGroup>
   <BuildConfigurations>
-    [BuildConfiguration]
+    [BuildConfiguration1];
+    [BuildConfiguration2];
+    ...
   </BuildConfigurations>
 <PropertyGroup>
 ```
 
 - `BuildConfiguration -> $(Configuration)|$(Platform)`
 - `$(Configuration) -> $(OSGroup)-$(TargetGroup)-$(ConfigurationGroup)`
 
+Example:
+```
+Pure netstandard configuration:
+<PropertyGroup>
+  <BuildConfigurations>
+    netstandard;
```

Comment:
We could but I was trying to eliminate the default "any" configurations for needing to duplicated everywhere. If we find that it is useful for some reason we can make more things explicit.